list/LIST_CONSULTATION_README.md

```diff
@@ -0,0 +1,211 @@
+- identifier
+- source
+- note
+- entry.flag
+- entry.deleted
+- entry.date
+
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "List",
+        "id": "1E8A8448-A0C1-11ED-808B-AC162D1F16F0-CONS",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-List-1"
+            ]
+        },
+        "status": "current",
+        "mode": "snapshot",
+        "title": "Surgery Consultation Note",
+        "code": {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "325851000000107",
+                    "display": "Consultation"
+                }
+            ]
+        },
+        "subject": {
+            "reference": "Patient/14013417-5eb8-4fb2-9916-4c1621e2533b"
+        },
+        "encounter": {
+            "reference": "Encounter/1E8A8448-A0C1-11ED-808B-AC162D1F16F0"
+        },
+        "date": "2010-12-16",
+        "orderedBy": {
+            "coding": [
+                {
+                    "system": "http://hl7.org/fhir/list-order",
+                    "code": "system",
+                    "display": "Sorted by System"
+                }
+            ]
+        },
+        "entry": [
+            {
+                "item": {
+                    "reference": "List/5a8d2ec6-807d-4db5-b6c0-a757bbfb5372"
+                }
+            }
+        ]
+    }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A `List (Consultation)` is mapped from references `resource` with a `resource.type` of `list` when mapping an [Encounter](../encounters/README.md) to an `EhrComposition` from a GP Connect FHIR `Encounter`
+
+| Mapped to (XML HL7 Encounter)                   | Mapped from (JSON FHIR / other source )                                                                           |
+|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| EhrComposition / Component [@typeCode="COMP"]`  | `resource` when the list contains a reference to `Encounter.id` and 'resource.resourceType` = `List` <sup>1</sup> |
+
+1.  For each reference in `resource.entry.item.reference`, the resultant topic list are parsed and are mapped to a `Component` when the referenced resource `resource.resourceType` is one of the following clinic records:
+
+* [AllergyIntolerance](../allergy%20intolerances/README.md)
+* [Condition](../conditions/README.md)
+* [DocumentReference](../document%20references/README.md)
+* [Immunization](../immunisations/README.md)
+* [MedicationRequest](../medication%20requests/README.md)
+* [Observation](../observations/README.md)
+* [ProcedureRequest](../procedure%20requests/README.md)
+* [ReferralRequest](../referral%20requests/README.md)
+* [DiagnosticReport](../diagnostic%20reports/README.md)
+
+<details><summary>Example XML</summary>
```

Comment:
Discussed the idea that removing the XML generated because of the TOPIC, and instead pointing
```
...this is where some stuff would go...
```